e2e/wdio/axe/report.acs-spec.ts

```diff
@@ -0,0 +1,37 @@
+import * as axe from 'axe-core';
+import { createHtmlReport } from 'axe-html-reporter';
+import { execute, open, pause, runAxeReport, waitForPresent } from '../driver/wdio';
+import { appURLs } from './app-url';
+import { BaseComponentPo } from '../platform/pages/base-component.po';
+
+describe('Accessibility test', () => {
+    it('Should check for accessibility issues using Axe', () => {
+        const { title } = new BaseComponentPo();
+        const options = { runOnly: { type: 'tag', values: ['wcag2a', 'wcag2aa'] } };
+        console.log(browser.capabilities.browserVersion);
+        for (const pageUrl in appURLs) {
+            for (const urls in appURLs[pageUrl]) {
+                open(`/fundamental-ngx#/${pageUrl}${appURLs[pageUrl][urls]}`);
+                pause(800);
```

Comment:
is pause needed? shouldn't waitForPresent(title) be enough?

Reply:
done.